host-configs/juno-armv8-gcc@5.2.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019-2021 Lawrence Livermore National Security, LLC and other
+# Variorum Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: MIT
+
+# c compiler
+set(CMAKE_C_COMPILER "/usr/bin/gcc" CACHE PATH "")
+
+# cpp compiler
+set(CMAKE_CXX_COMPILER "/usr/bin/g++" CACHE PATH "")
+
+# fortran compiler
+# set(CMAKE_Fortran_COMPILER  "/usr/bin/gfortran" CACHE PATH "")
```

Comment:
Let's replace these commented out lines for setting the fortran compiler with ENABLE_FORTRAN: `set(ENABLE_FORTRAN OFF CACHE BOOL "")`

Reply:
This is much better.